Realm/RLMArray.h

```diff
@@ -175,6 +175,18 @@ RLM_ASSUME_NONNULL_BEGIN
  */
 - (void)replaceObjectAtIndex:(NSUInteger)index withObject:(RLMObject *)anObject;
 
+/**
+ Exchanges the objects in the array at given indexes.
+
+ Throws an exception when either index is greater than the number of objects in this RLMArray.
```

Comment:
Pretty sure this was all copied verbatim from NSArray, but I'll make the changes